src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/ApplicationDeployer.cs

```diff
@@ -134,7 +137,7 @@ protected void AddEnvironmentVariablesToProcess(ProcessStartInfo startInfo)
 
             SetEnvironmentVariable(environment, "ASPNETCORE_ENVIRONMENT", DeploymentParameters.EnvironmentName);
```

Comment:
Doesn't affect the publish process since this is pretty AspNet specific. Plus it avoids duplicating this in two places.